Image  in
---
Ints pt_min
Ints pt_max
Image debug_image
Floats confidence
geometry_msgs/Pose[] poses  
